### ISSUE TITLE
[Feature] Can you rebuild bitrate settings?

### ISSUE THREAD

**Issue by Zuko** [acomminos/Plumble-Legacy#52]
My ideas:
- add 3 predefined settings (like in desktop app)
- and "something like" amplifier settings for manual adjust

**Comment by acomminos** [comment]
Definitely. I'd like to offer much better control over audio input and output, as well as adding echo processing so Plumble can be used without a headset.
